docs/structured-streaming-programming-guide.md

```diff
@@ -2103,7 +2103,7 @@ streamingDf.dropDuplicates("guid")
 // With watermark using guid and eventTime columns
 streamingDf
   .withWatermark("eventTime", "10 seconds")
-  .dropDuplicates("guid", "eventTime")
+  .dropDuplicates(Vector("guid", "eventTime"))
```

Comment:
Scala has `dropDuplicates(col1: String, cols: String*)`?

Reply:
Yeah make sense. Will revert it.